docs/deployment.md

```diff
@@ -0,0 +1,34 @@
+Deployment
+==========
+
+In order to follow our "Service First" guiding principal, we deploy Apex continuously as a service on [OperateFirst](https://operate-first.cloud) using GitOps.
```

Comment:
You could add a footnote on this guiding principle in the main README that links to this page for how we're putting it into practice.